src/odemis/acq/stream/_helper.py

```diff
@@ -90,29 +90,27 @@ def __init__(self, name, detector, dataflow, emitter, scanner=None, sstage=None,
                                          range=((0, 0, 0, 0), (1, 1, 1, 1)),
                                          cls=(int, long, float),
                                          setter=self._setROI)
+
+        # Start with pixel size to fit 1024 px horizontally (and adjust for the hardware)
+        epxs = self._scanner.pixelSize.value
```

Comment:
It would be easier to understand with a short comment what all these values are.